src/uTensor/ops/ArgMinMax_kernel.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef UTENSOR_ARG_MIN_MAX_KERNEL_H
+#define UTENSOR_ARG_MIN_MAX_KERNEL_H
+
+#include "context.hpp"
+#include "types.hpp"
+#include "tensor.hpp"
+#include "uTensor_util.hpp"
+
+namespace uTensor {
+enum ArgMinMaxCompareFlag {
+  Max = 1,
+  Min = -1,
+};
+
+template <typename Tin>
+void arg_min_max_kernel(Tensor &output, Tensor &input, Tensor &axis, ArgMinMaxCompareFlag cmp = Max) {
```

Comment:
If input is not modified it should be a `const Tensor& input`
Also please, place the references next to the types and not the var names to fit the style of the rest of the code. :)

Reply:
Ok.